sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1705,6 +1704,12 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				currentVersion = 236;
 				getDao(Contact.class).executeRaw("ALTER TABLE contacts ADD COLUMN returningTraveler varchar(255);");
 
+			case 237:
+				currentVersion = 237;
```

Comment:
Please remove the not null constraint; SQLite doesn't support the removal of constraints in retrospection so we try to avoid it in the mobile app. Also please remove the default value because I suspect that this is also not editable afterwards